contrib/rpm/buildah.spec

```diff
@@ -240,7 +240,7 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 
 * Wed Sep 11, 2019 Tom Sweeney <tsweeney@redhat.com> 1.11.1-1
 - Add --devices flag to bud and from
-- Downgrade .papr to highest atomic verion
+- Downgrade .papr to highest atomic version
```

Comment:
I'd rather not change these lines in here, the  changelog.txt or the  CHANGELOG.md file.  The right or wrong, the commits went in with these mis-spellings and if we change them here, it will be harder to find them 2 years down the road.

Reply:
The problem is this becomes I will have to eliminate buildah.spec from the check like I have CHANGELOG.md